lib/Sema/TypeCheckType.cpp

```diff
@@ -785,6 +785,36 @@ static Type diagnoseUnknownType(TypeChecker &tc, DeclContext *dc,
   }
 
   // Qualified lookup case.
+
+  // Try ignoring access control.
+  NameLookupOptions lookupOptions = defaultMemberLookupOptions;
```

Comment:
Since this is repeated twice, do you think it makes sense to define a new memberTypeLookupOptions or similar?